apps/studio/components/interfaces/Auth/Users/UsersV2.tsx

```diff
@@ -54,6 +55,9 @@ import { formatUserColumns, formatUsersData } from './Users.utils'
 import { UsersFooter } from './UsersFooter'
 import { UsersSearch } from './UsersSearch'
 
+// [Joshen] Arbitary threshold value for what's considered to be a lot of users
+const SORT_BY_VALUE_COUNT_THRESHOLD = 10_000
```

Comment:
nit: Don't think the comment is needed, variable defines it fine

```suggestion
const SORT_BY_VALUE_COUNT_THRESHOLD = 10_000
```